src/WallToolPaths.cpp

```diff
@@ -73,12 +73,13 @@ const VariableWidthPaths& WallToolPaths::generate()
     {
         const coord_t wall_transition_length = settings.get<coord_t>("wall_transition_length");
         const Ratio wall_transition_threshold = settings.get<Ratio>("wall_transition_threshold");
+        const double wall_distribution_count = settings.get<int>("wall_distribution_count");
```

Comment:
The wall distribution count is converted from an `int` to a `double` and later on when the `makeStrategy` is called, it is converted to a `float`.

Reply:
https://github.com/Ultimaker/CuraEngine/pull/1477/commits/50bd89aa5c032f18b837cfa895e50740fef89cf0